src/code/speed_meter.c

```diff
@@ -186,25 +186,26 @@ void SpeedMeter_DrawAllocEntries(SpeedMeter* meter, GraphicsContext* gfxCtx, Gam
     }
 
     thga = (TwoHeadGfxArena*)&state->tha;
-    SpeedMeter_InitAllocEntry(&entry, thga->size, thga->size - THA_GetSize((TwoHeadArena*)thga),
+    //! @bug THA_GetRemaining call should be THGA_GetRemaining like the others below, harmless as-is
+    SpeedMeter_InitAllocEntry(&entry, thga->size, thga->size - THA_GetRemaining(&thga->tha),
```

Comment:
I argue that it is a bug since suppose the functionality of `THGA_GetRemaining` were extended in any way, this could be a harmful difference. I think it's useful to point this out as a bug comment